src/kakarot/library.cairo

```diff
@@ -71,6 +71,7 @@ namespace Kakarot {
     // @param value The value of the execution
     // @param gas_limit The gas limit of the execution
     // @param gas_price The gas price for the execution
+    // @param reverted Whether the transaction is reverted or not
```

Comment:
iirc execution specs (eels) returns the whole evm object after execute, 
why did we decide to return a subset of the execution context?

Reply:
no idea, but I don't think it's worth changing this rn

Reply:
I don't remember, but if we don't return the whole ExecutionContext object, the summary does have stack, memory and return_data, gas_used and status. It seems all we need no?